core/src/main/java/org/testcontainers/containers/Container.java

```diff
@@ -174,6 +175,15 @@ default SELF withFileSystemBind(String hostPath, String containerPath) {
      */
     SELF withCopyFileToContainer(MountableFile mountableFile, String containerPath);
 
+    /**
+     * Set the file to be copied before starting a created container
```

Comment:
```suggestion
     * Set the Transferable to be copied before starting a created container
```

It is not really a file, or?